tangelo/algorithms/variational/tests/test_adapt_vqe_solver.py

```diff
@@ -14,7 +14,13 @@
 
 import unittest
 
+import numpy as np
+
 from tangelo.algorithms.variational import ADAPTSolver
+from tangelo.linq import Circuit, Gate
```

Comment:
could we tidy things up a bit ?

molecule lib, then linq, then all the toolboxes, maybe?